controllers/usersignup/space_test.go

```diff
@@ -32,7 +33,7 @@ func TestNewNsTemplateSetSpec(t *testing.T) {
 		nsTemplateTier := newNsTemplateTier("advanced", "dev", "stage", "extra")
 
 		// when
-		setSpec := NewNSTemplateSetSpec(nsTemplateTier)
+		setSpec := space.NewNSTemplateSetSpec(nsTemplateTier)
 
 		// then
 		assert.Equal(t, newExpectedNsTemplateSetSpec(), setSpec)
```

Comment:
these `TestNewNsTemplateSetSpec` tests could have been moved to `space` package since the `NewNSTemplateSetSpec` is located there

Reply:
🤯

Reply:
done in 479ea8d